app/assets/stylesheets/ctl_base_ui/notices.scss

```diff
@@ -39,3 +39,14 @@
   color: $white;
   background-color: $dark_red;
 }
+
+div[class*='notice-'] {
```

Comment:
if you put this in the .notice-default I guess it wouldn't cascade down, because it's nested? I don't have any problems with this approach though.